### ISSUE TITLE
Try to check __init__.py file

### ISSUE THREAD

**Issue by centelles** [antonioIrizar/django-check-migration-wtf#41]
Sometimes `django-check-migration-wtf` try to check `__init__.py` file inside of the `migrations` directory. Example:

```bash
App name: trse- Migration: 0001_initial
    Migration is secure.
Traceback (most recent call last):
  File "manage.py", line 8, in <module>
    execute_from_command_line(sys.argv)
  File "/opt/hostedtoolcache/Python/3.8.2/x64/lib/python3.8/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
    utility.execute()
  File "/opt/hostedtoolcache/Python/3.8.2/x64/lib/python3.8/site-packages/django/core/management/__init__.py", line 375, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/opt/hostedtoolcache/Python/3.8.2/x64/lib/python3.8/site-packages/django/core/management/base.py", line 323, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/opt/hostedtoolcache/Python/3.8.2/x64/lib/python3.8/site-packages/django/core/management/base.py", line 364, in execute
    output = self.handle(*args, **options)
  File "/opt/hostedtoolcache/Python/3.8.2/x64/lib/python3.8/site-packages/django_check_migration_wtf/management/commands/check_migration.py", line 38, in handle
    self.evaluate_migration(app_name, migration_name)
  File "/opt/hostedtoolcache/Python/3.8.2/x64/lib/python3.8/site-packages/django_check_migration_wtf/management/commands/check_migration.py", line 56, in evaluate_migration
    migration = self.loader.get_migration_by_prefix(app_label, migration_name)
  File "/opt/hostedtoolcache/Python/3.8.2/x64/lib/python3.8/site-packages/django/db/migrations/loader.py", line 144, in get_migration_by_prefix
    raise KeyError("There no migrations for '%s' with the prefix '%s'" % (app_label, name_prefix))
KeyError: "There no migrations for 'trse' with the prefix '__init__'"
##[error]Process completed with exit code 1.
```

**Comment by jenarvaezg** [comment]
WTF

**Comment by antonioIrizar** [comment]
@centelles thanks about the report. It was fixed with release 1.0.1
